2-mandatory/6-journey-planner.js

```diff
@@ -121,8 +125,13 @@ function getLocationName() {}
 
   Advanced challange: try to use arrow function when invoking an array method.
 */
+
 function journeyPlanner(locations, transportMode) {
-  // Implement the function body
+  const accessibleLocations = locations
+    .filter((place) => place.includes(transportMode))
+    .map((place) => place[0]); // Extracting only the location names
+
+  return accessibleLocations;
```

Comment:
This solution works, but try to change it using your functions above, so that journeyPlanner would call getLocationName and isAccessibleByTransportMide